recipes/sdl/3.x/conanfile.py

```diff
@@ -0,0 +1,450 @@
+from conan import ConanFile, conan_version
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, replace_in_file, rm, rmdir, copy
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout, CMakeDeps
+from conan.tools.env import Environment
+
+import os
+
+required_conan_version = ">=1.55.0"
+
+# Subsystems, CMakeLists.txt#L234
+_subsystems = [
+    ("audio", []),
+    ("video", []),
+    ("gpu", ["video"]),
+    ("render", ["video"]),
+    ("camera", ["video"]),
+    ("joystick", []),
+    ("haptic", ["joystick"]),
+    ("hidapi", []),
+    ("power", []),
+    ("sensor", []),
+    ("dialog", []),
+]
+
+class SDLConan(ConanFile):
+    name = "sdl"
+    description = "A cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware"
+    license = "Zlib"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.libsdl.org"
+    topics = ("sdl3", "audio", "keyboard", "graphics", "opengl")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        **{
+            "shared": [True, False],
+            "fPIC": [True, False],
+        },
+        **{subsystem: [True, False] for subsystem, _ in _subsystems},
+        **{
+            "alsa": [True, False],
+            "pulseaudio": [True, False],
+            "sndio": [True, False],
+            "opengl": [True, False],
+            "opengles": [True, False],
+            "x11": [True, False],
+            "xcursor": [True, False],
+            "xinerama": [True, False],
+            "xinput": [True, False],
+            "xrandr": [True, False],
+            "xscrnsaver": [True, False],
+            "xshape": [True, False],
+            "xvm": [True, False],
+            "wayland": [True, False],
+            "vulkan": [True, False],
+            "metal": [True, False],
+            "directx": [True, False],
+            "libudev": [True, False],
+            "dbus": [True, False],
+            "libusb": [True, False],
+        }
+    }
+
+    default_options = {
+        **{
+            "shared": False,
+            "fPIC": False,
```

Comment:
```suggestion
            "fPIC": True,
```
Probably wasn't intentional?